cli.js

```diff
@@ -36,6 +36,7 @@ const cli = meow(`
 	  --authentication         Credentials for HTTP authentication
 	  --debug                  Show the browser window to see what it's doing
 	  --overwrite              Overwrite the destination file if it exists
+	  --launch-options         Puppeteer launch options as JSON
```

Comment:
This needs to be reflected in the readme too.

Reply:
Done